src/ducks/resource.js

```diff
@@ -133,32 +91,34 @@ function createTranslation(original, translations, type, language) {
     type = 'tutorials';
   }
   return (dispatch) => {
-    const newResource = Object.assign({}, original);
-    delete newResource.id;
-    delete newResource.href;
-    delete newResource.created_at;
-    delete newResource.updated_at;
-    delete newResource.links.attached_images;
-    newResource.language = language.value;
+    const newResource = {
+      translated_type: original.translated_type,
```

Comment:
the `translated_type` & `translated_id` params should be outside the resource payload and specified as query params on the resource URL. e.g. `/api/translations?translated_type=Project&translated_id=1`

Reply:
zooniverse/json-api-client#40 should take care of this. This PR will need updating to use the new client, once it's ready.